src/components/Article.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useEffect, useState } from "react";
+import { useParams } from "react-router-dom";
+import { fetchArticle } from "../utils/api";
+import CardHeader from "./CardHeader";
+import CardFooter from "./CardFooter";
+
+function Article() {
+  const [article, setArticle] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
+
+  const { id } = useParams();
+
+  useEffect(() => {
+    fetchArticle(id).then((article) => {
+      setArticle(article);
+      setIsLoading(false);
+    });
+  }, [id]);
+
+  if (isLoading) return <p>Getting your article...</p>;
+  else
+    return (
+      <div className="article-container ">
+        <CardHeader
+          author={article.author}
+          created_at={article.created_at}
+          topic={article.topic}
+        />
+        <main>
+          <h2>{article.title}</h2>
+          <p>{article.body}</p>
+        </main>
+        <CardFooter
+          votes={article.votes}
+          comment_count={article.comment_count}
+        />
+      </div>
+    );
+}
```

Comment:
I'm absolutely fine with your CardHeader CardFooter namings but if you wanted to be more verbose ArticleCardHeader/ArticleCardFooter would give a little more information about what these cards relate to

Reply:
Thanks Sarah! I have reused them for my comment headers and footers cheekily so I will leave it as it is :) I would definitely otherwise have changed it! thank you